utils/components.py

```diff
@@ -298,3 +301,302 @@ def ilab_importer_op(repository: str, release: str, base_model: dsl.Output[dsl.M
             f"ilab --config=DEFAULT model download --repository {repository} --release {release} --model-dir {base_model.path}"
         ],
     )
+
+
+@dsl.component(base_image=RUNTIME_GENERIC_IMAGE)
+def test_judge_teacher_models(secret_name: str):
```

Comment:
I'd prefer this name or similar because the current name sounds like it checks both in one execution of the function.
```suggestion
def test_model_connection(secret_name: str):
```